packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRowAction.tsx

```diff
@@ -53,26 +52,16 @@ export function TransactionsTableRowAction({
   const l2NetworkName = getNetworkName(l2Network.id)
   const networkName = type === 'deposits' ? l1NetworkName : l2NetworkName
 
-  const chainId = useChainId()
   const { claim, isClaiming } = useClaimWithdrawal()
   const { claim: claimCctp, isClaiming: isClaimingCctp } = useClaimCctp(tx)
   const { isConfirmed } = useRemainingTime(tx)
 
-  const { isEthereumMainnetOrTestnet, isArbitrum } = isNetwork(chainId)
-
   const currentChainIsValid = useMemo(() => {
```

Comment:
We still have to check against the connected network. Let's use `useNetwork` instead of `useChainId` though